generateData/sampleData/generateData.js

```diff
@@ -0,0 +1,297 @@
+/*
+*
+*  For Dummy Data:
+*  Send a request to this api, and specify the number of users you want.
```

Comment:
Is it better to make an axios request, rather than having to copy the data into a new file?